docs/getting-started/github-apps/snippet--installation.rst

```diff
@@ -33,10 +33,24 @@ You should now see the organization and the repositories you chose to keep track
 .. figure:: /assets/images/docs/my-services-filled.png
 
 
-A note on permissions
-~~~~~~~~~~~~~~~~~~~~~
+A note on permissions when installing the GitHub App to a GitHub organization
```

Comment:
I suggest "installing a GitHub App" or "installing the Dockstore GitHub App", with a ripple effect of how you refer to it in the next section, if you change it.